### ISSUE TITLE
How to output logger (class name) with newrelic json layout?

### ISSUE THREAD

**Issue by apuchkov** [newrelic/newrelic-logenricher-dotnet#96]
Hello,

When using NLog you typically include a class name from which messages are logged by getting a class logger like:
```csharp
private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
```
Then logger name (class name) can be included in the output:
```xml
<target xsi:type="File" name="f1" fileName="app.log" layout="${longdate}||${level}|${logger}|${message}" />
```
producing a resulting logging line like:
```
2020-12-29 12:45:22.1902|TRACE|RequestTracer|Request completed.
```
It helps to get a better understanding of where log messages are coming from. It also allows filtering of log messages by class name.

Here's a log line produced by the newrelic json layout. It does not include logger name. Is there a way to include logger name as a separate property?
```json
{
"timestamp":1609263922190,
"log.level":"TRACE",
"message":"Request completed.",
"message.template":"Request completed.",
"thread.id":"149",
"correlation.id":"ce304a5d-206b-4b01-b2a5-550ead04f010",
"process.id":"35396"
}
```

**Comment by nr-ahemsath** [comment]
Hello @apuchkov,

Thanks for submitting a feature request.  Please see this PR that I believe adds the functionality you want: https://github.com/newrelic/newrelic-logenricher-dotnet/pull/97

**Comment by apuchkov** [comment]
That's exactly what I was looking for! Thanks for such a fast turn around.

**Comment by nr-ahemsath** [comment]
PR #97 has been merged and released in version v1.1.0 of our libraries for NLog and Log4Net.  Closing.
